apps/web/package.json

```diff
@@ -73,7 +73,6 @@
     "@storybook/addon-essentials": "^6.4.14",
     "@storybook/addon-links": "^6.4.14",
     "@storybook/node-logger": "^6.4.14",
-    "@storybook/preset-create-react-app": "^4.0.0",
```

Comment:
@ainouzgali is this not needed anymore? From what I remember it's need to render the react storybook

Reply:
i reverted this :) you are right